src/switch_rtp.c

```diff
@@ -1353,7 +1353,6 @@ SWITCH_DECLARE(void) switch_srtp_err_to_txt(srtp_err_status_t stat, char **msg)
 	else if (stat == srtp_err_status_read_fail) *msg="couldn't read data";
 	else if (stat == srtp_err_status_write_fail) *msg="couldn't write data";
 	else if (stat == srtp_err_status_parse_err) *msg="error parsing data";
-	else if (stat == srtp_err_status_write_fail) *msg="couldn't read data";
```

Comment:
this change is scheduled to be merged as part of another commit. the other changes lgtm.